pkg/route/api/validation/validation.go

```diff
@@ -277,3 +281,24 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) (*x509.
 
 	return cert, nil
 }
+
+// validateWildcardPolicy tests that the wildcard policy is either empty or one of the supported types.
+func validateWildcardPolicy(policy routeapi.WildcardPolicyType, fldPath *field.Path) *field.Error {
+	// Check policy value if specified (empty is ok).
+	if len(policy) == 0 {
```

Comment:
It still happens - if the tests create a Route object without specifying that field value `wildcardPolicy`. Maybe the code in defaults.go is not right - dunno. I kept this as the validation tests fail otherwise.